src/workerd/api/actor-state.c++

```diff
@@ -660,6 +660,22 @@ jsg::Promise<jsg::Value> DurableObjectStorage::transaction(jsg::Lock& js,
   });
 }
 
+jsg::Promise<void> DurableObjectStorage::flush(v8::Isolate* isolate) {
```

Comment:
Should this take `js::Lock& js` instead of `v8::Isolate* isolate`?

Reply:
Yep, I think that would be strictly better! (We have to take an isolate for `transformMaybeBackpressure()` in other functions, but not for flush.)